### ISSUE TITLE
Household and Household Member: locations questions use old UI style

### ISSUE THREAD

**Issue by gmcressman** [ICTatRTI/coconut-mobile-plugin-zanzibar#8]
The location options in the Household and Household Member forms appear to using the previous UI style, rather than MD:
![image](https://user-images.githubusercontent.com/329768/27753219-ce57b436-5db2-11e7-8300-b5e03ec78857.png)

**Comment by scloo** [comment]
Tried to switch to the mdl radio buttons but it broke something. Not able to type anything as input after that. I did make the size smaller for now.
![Screen Shot 2017-07-07 at 3.38.24 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/571a5f9b500b7b0e00eca657/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c57e0b77246454f46b7191a37bf111234f5d5b5ecc3c4a4356bcbbd671f3e57862804e0ee4d4395d21c13d45fab1871bb701ba738f1a86bebf8c1ad2f42f67b6d5db1063474f4dcbed5727a39841c95cff42e54853a7113461e5712431.png)
